_integration-schemas/3plcentral/customers.md

```diff
@@ -0,0 +1,489 @@
+---
+tap: "3plcentral"
+version: "1.0"
+
+name: "customers"
+doc-link: "http://api.3plcentral.com/rels/"
+singer-schema: "https://github.com/singer-io/tap-3plcentral/blob/master/tap_3plcentral/schemas/customers.json"
+description: "This table contains information about customers."
+
+replication-method: "Full Table"
+
+api-method:
+    name: "REL documentation"
+    doc-link: "http://api.3plcentral.com/rels/customers/customers"
```

Comment:
For `name`, use the same title that's used in the provider's API docs.

If there isn't one (like in this case), you can use the API method (ex: `GET`) + whatever the object is (`customers`). So the name could be `Get customers`.